doc/data/messages/b/bad-docstring-quotes/bad.py

```diff
@@ -0,0 +1,4 @@
+# fmt: off
+def square_root():
+    '''Return the square root of input.'''  # [bad-docstring-quotes]
+# fmt: on
```

Comment:
```suggestion
def square_root():
    "Return the square root of input."  # [bad-docstring-quotes]
```

Reply:
😄 Wow I don't know that will work, thanks

Reply:
Well that works but it's not really a docstring, just a piece of code that does nothing thus the message 😄. It's like 
```python
1 # Could be 'a = 1' and be useful
```
Just a declaration that does nothing.

Reply:
FYI both of these examples are actually valid as docstrings.
```python
>>> def foo():
...     "I am a docstring"
...
>>> foo.__doc__
'I am a docstring'
```
```python
>>> def foo():
...     '''I am a docstring'''
...
>>> foo.__doc__
'I am a docstring'
```
This more of a style check than functionality 😄

Reply:
> Just a declaration that does nothing.

This is actually true if someone tries to use an f-string as a docstring though 😄 

```python
>>> def foo():
...     f"""f-strings can't be docstrings"""
...
>>> foo.__doc__
>>>
```

We have a check in `flake8-bugbear` that warns about this if the first statement is a pointless f-string statement since you can assume that the user probably wanted to use it as a docstring. Potentially could add that as another future pylint enhancement

Reply:
Hmm, interesting, I did not know. The three quotes standard is so prevalent that I did not even try !

We do have a ``W0104: Statement seems to have no effect (pointless-statement)``  for 
```
def f():
    f'''' ''''
````
 and  nothing for 
```
def f():
    " "
````
So a contributor knew at some point 😄

Reply:
yeah I had the same reaction when I found out, I only know about it because of that f-string check 😄 

> So a contributor knew at some point 😄

It's because the quotes in that example go in the `doc_node` attribute in the AST so likely the writer of W0104 had no idea haha